Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -154,7 +155,8 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
     u32 pad0;
   } req;
 
-  req.command = Memory::Read_U32(_BufferIn + 0);
+  // Ignore the first two bits
```

Comment:
The thing is I wrote this code months ago and I didn't add a note then.
Spec says the commands are 6 bits, but I honestly don't remember what the other two bits are for.
What I do know is this caused certain commands to be misinterpreted or not interpreted at all.